CHANGELOG.md

```diff
@@ -0,0 +1,6 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+## [Unreleased]
+
+* Lots of stuff way to complicate for my drunk ass
```

Comment:
Can you be a little less aggressive here ?
There is other ways to say you are dumb you know ?